### DISCUSSION TITLE
[✨] Revalidation data in useloader after useAction runs

### DISCUSSION THREAD

**Discussion by Adbib** [QwikDev/qwik-evolution#125] [Proposals For Qwik]
### Is your feature request related to a problem?

I have the Loader get me a list of data, when I use an Action to delete an item from the list the UI still has old data from the Loader

### Describe the solution you'd like

a way to revalidate data in Loader after submitting an Action 

### Describe alternatives you've considered

In Remix for example, after you use an Action the Loader gets automatically revalidated, also you can control if you should revalidate or not using the shouldRevalidate function .

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
